src/error.rs

```diff
@@ -102,3 +112,38 @@ impl DeadlineExceeded {
         Self { deadline, now }
     }
 }
+
+/// `stderr` output of the browser child process
+///
+/// This implements a custom `Debug` formatter similar to [`std::process::Output`]. If the output
+/// is valid UTF-8, format as a string; otherwise format the byte sequence.
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct BrowserStderr(Vec<u8>);
```

Comment:
I was wondering but, I don't think it is worth converting this to a `std::error::Error`, since it is only used in the `launch` error variant, which is the actual error.

so this is fine.